### DISCUSSION TITLE
claro, a GitHub Classroom CLI for teachers

### DISCUSSION THREAD

**Discussion by emersonmello** [community/community#18978] [GitHub Education]
Hi,

I wrote [**claro**](https://github.com/emersonmello/claro) a cli tool that offers a simple interface that allows the teacher to clone all student repositories at once for grading and then send grades at once to all these repositories. 

**claro** was inspired by the [Git-Gud-tool](https://github.com/NikolaiMagnussen/Git-Gud-tool) and it was created to make my life simpler as a teacher who relies on Github Classroom. It is suitable for scenarios where the teacher needs to manually grade each assignment and the Github Classroom autograding is not an option.

There are binaries available for macOS and Linux. 

https://github.com/emersonmello/claro/

PS: **claro** was written in Golang, so it is quite easy to generate binary for Windows (i.e. `GOOS=windows GOARCH=amd64 go build`). I didn't because I don't have Windows to test it.

**Comment by Ajjig** [comment]
Thats sounds interesting
